kyaml/fn/framework/gosdk/krmfunction.go

```diff
@@ -0,0 +1,7 @@
+package gosdk
+
+import (
+	"context"
+)
+
+type KRMFunction func(ctx context.Context, rl *ResourceList) error
```

Comment:
What is the purpose of passing in a context?

Reply:
Dropped.
Earlier I thought we may want to add timeout.